### DISCUSSION TITLE
AvaloniaUI Web assembly publish size is too bigger than stand blazor web assembly project

### DISCUSSION THREAD

**Discussion by felixf4xu** [AvaloniaUI/Avalonia#17491] [Q&A]
### Describe the bug

I created 2 web assembly project, one is blazor web assembly, by `dotnet new blazorwasm -o BlazorWasm`, the other is avaloniaui web browser project, by `dotnet new avalonia.xplat -o AvaloniaX`.

Both are using this configurations to `dotnet publish`:
```xml
    <RunAOTCompilation>true</RunAOTCompilation>
    <WasmStripILAfterAOT>true</WasmStripILAfterAOT>
    <InvariantGlobalization>true</InvariantGlobalization>
    <InvariantTimezone>true</InvariantTimezone>
```

Then I check the publish folder:

avaloniaui web browser project:
>dotnet.native....wasm: 28.3M
dotnet.native....wasm.br: 6.1M 

blazor web assembly project:
>dotnet.native....wasm: 11.1M
dotnet.native....wasm.br: 2.5M 

Why the size is so bigger than blazor web assembly project?

### To Reproduce

`dotnet new avalonia.xplat -o AvaloniaX`

### Expected behavior

dotnet.native.wasm should be similar to blazor wasm project

### Avalonia version

11.1.0

### OS

WebAssembly

### Additional context

.net9

**Comment by maxkatz6** [comment]
Likely pre-included font files take a lot of this space. Avaloina.Fonts.Inter package to be specific.
But also, these should be compared file by file, without AOT first.
